style.css

```diff
@@ -1,4 +1,88 @@
 
+/* General Style */
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* header style */
+
+.header{
+    padding: 32px ;
+    background-color: #35424a;
+    color: white;
+    border-bottom: 3px solid #e8491d;
+
+}
+#header{
+    color: #e8491d;
```

Comment:
the colour is a little bit uncomfortable,
 I tried to use another colour, and it looks good  `#36a080`